recipe/meta.yaml

```diff
@@ -28,12 +27,11 @@ requirements:
     - cmake
     - bison       # [not win]
     - flex        # [not win]
```

Comment:
Maybe not needed

Reply:
My experience is the build fails without them